man/ocaml.1

```diff
@@ -37,23 +37,19 @@ command is the toplevel system for OCaml,
 that permits interactive use of the OCaml system through a
 read-eval-print loop. In this mode, the system repeatedly reads OCaml
 phrases from the input, then typechecks, compiles and evaluates
-them, then prints the inferred type and result value, if any. The
+them, then prints the inferred type and result value, if any.
 system prints a # (hash) prompt before reading each phrase.
```

Comment:
This diff looks weird, is there an issue here? (`The` is gone?)

Reply:
In fact I meant to remove the whole sentence.